### PR TITLE
Regenerated schema based on Swagger and added test.

### PR DESCRIPTION
**Pull request by huizlAzure**

### CODE REVIEW

File: schemas/2018-07-01/Microsoft.Storage.json [hunk 1]
```diff
@@ -105,6 +115,64 @@
       ],
       "description": "Microsoft.Storage/storageAccounts"
     },
+    "storageAccounts_blobServices": {
```
Comment (TianoMS) [inline schemas/2018-07-01/Microsoft.Storage.json]: >storageAccounts_blobServices [](start = 5, length = 28)

do you need to add this to 2015-01-01/deploymentTemplate.json and 2014-04-01-preview/deploymentTemplate?

Comment (huizlAzure) [reply schemas/2018-07-01/Microsoft.Storage.json]: Blob services APIs are supported only in 2018-07-01 and later SRP API versions, so I think we do not want to add it to those versions here?

I actually am not sure why all the existing objects have 2015-01-01 in 'oneOf', on the side of the type ref. I'm new to Azure schema. Any comments/suggestions?

Comment (huizlAzure) [reply schemas/2018-07-01/Microsoft.Storage.json]: Tiano: I've added BlobServices in those deployment template files in latest commit. Kindly take a look.

File: tests/2018-07-01/Microsoft.Storage.tests.json [hunk 2]
```diff
@@ -197,6 +197,67 @@
           "immutabilityPeriodSinceCreationInDays": 3
         }
       }
+    },
+    {
+      "name": "storageAccountBlobServices",
```
Comment (TianoMS) [inline tests/2018-07-01/Microsoft.Storage.tests.json]: >storageAccountBlobServices [](start = 15, length = 26)

test case is failing.

File: tests/2018-07-01/Microsoft.Storage.tests.json [hunk 3]
```diff
@@ -197,6 +197,67 @@
...
+                "allowedMethods": [
+                  "GET",
+                  "PUT"
+                ],
+                "maxAgeInSeconds": 500,
+                "exposedHeaders": [
+                  "x-ms-meta-customheader",
+                  "x-ms-meta-data*"
+                ],
+                "allowedHeaders: [
```
Comment (TianoMS) [inline tests/2018-07-01/Microsoft.Storage.tests.json]: >: [](start = 31, length = 1)

a double quote is missing here.

File: tests/2018-07-01/Microsoft.Storage.tests.json (hunk 2 above)
Comment (huizlAzure) [reply tests/2018-07-01/Microsoft.Storage.tests.json]: Fixed missing double quote. Where do I find test results in the PR?

**Pull request merged**